R/merge_sce_list.R

```diff
@@ -80,6 +85,16 @@ merge_sce_list <- function(sce_list = list(),
      Please check that `retain_coldata_cols` was correctly specified.")
   }
 
+  # If we are including altExps, extract them now for separate merging, to be
+  #  added back into the final merged SCE at the end
+  if (include_altexp) {
+    altexp_list <- sce_list |>
+      purrr::map(altExp)
```

Comment:
This will get the first `altExp`, so we are assuming that all the altExps across the merge are the same, which seems dangerous. We need probably want to use `altExpNames()` first to get the full list of altExps we can expect. Then we have to go through them one at a time. 

Alternatively, you could use `AltExps()` to get lists of altExps with names, but I think separating the tasks will be worth the effort.